plugins/woocommerce-blocks/assets/js/base/components/cart-checkout/cart-line-items-table/style.scss

```diff
@@ -156,4 +155,16 @@ table.wc-block-cart-items {
 			padding-right: $gap;
 		}
 	}
+
+	&.has-dark-controls {
+		.wc-block-cart-items {
+			border-color: $input-border-dark;
+			th {
+				color: $input-text-dark;
+			}
+			td {
+				border-color: $input-border-dark;
+			}
+		}
+	}
```

Comment:
This highlighted an issue in our approach and how easy it is to mess up dark mode. I think we should switch to using CSS variables for colors, and at top level, we do this:

```css
:root {
	--wc-checkout-input-border: $universal-border-light;
}

.has-dark-controls {
	--wc-checkout-input-border: $universal-border-dark;
}
```